routes/app/app-manager.js

```diff
@@ -40,8 +42,10 @@ exports.createApplication = function(req, res, next) {
     console.log(app);
     app.save(function(err) {
         if(err) {
+            logging.accessLogger(req.user._id,req.url,"user_activity", "The application could not be saved.",false, app);
             next(err);
         } else {
+            logging.accessLogger(req.user._id,req.url,"user_activity", "An application was successfully created.",true, app);
```

Comment:
@nick22891 shouldn't the `entity` parameter be empty and the `end_result` be app?